convert/README.md

```diff
@@ -0,0 +1,185 @@
+# Domino CDK to Terraform migration
+
+Domino is deprecating this CDK repository, and switching all future EKS
+provisioning work to our Terraform module, located here:
+
+https://github.com/dominodatalab/terraform-aws-eks
+
+The scripts in this directory will help you convert the management your
+CDK-provisioned Domino infrastructure to the above Terraform module.
+
+Your EKS cluster, EFS store, and S3 buckets will all remain in-tact.
+The original ASGs and any bastion used for the install will be replaced
+by ones generated in Terraform.
+
+## Contents
+
+* convert.py - The core migration script
+* README.md - This file
+* terraform/ - Terraform to take over CDK infrastructure
+* cloudformation-only/ - Terraform to create a special IAM role to remove the CDK CloudFormation stack
+
+## Migration steps
+
+To convert the CDK stack to terraform, we will...
+
+1. Create input variables into Terraform
+2. Generate a set of Terraform import commands to import various AWS resources into the new Terraform
+3. Initialize the Terraform directory and import the resources
+4. Evaluate the Terraform plan
+5. Apply the Terraform plan
+6. Scale down the old auto-scaling groups
+7. Validate Domino functionality
+8. Delete unneeded AWS resources
+9. Delete the old CloudFormation stack
+
+### convert.py usage
+
+    usage: convert.py [-h] {create-tfvars,resource-map,get-imports,delete-stack,print-stack} ...
+    
+    terraform eks module importer
+    
+    options:
+      -h, --help            show this help message and exit
+    
+    commands:
+      {create-tfvars,resource-map,get-imports,delete-stack,print-stack}
+	create-tfvars       Generate tfvars
+	resource-map        Create resource map for customization/debugging of get-imports command (optional step)
+	get-imports         Get terraform import commands
+	delete-stack        Get commands to delete old stack
+	print-stack         Print CDK stack resources
+
+### Create terraform variables
+
+First, we must create the variables going into our Terraform config.
+
+The following command will look at the current CDK stack and autopropagate the Terraform variables:
+
+    ./convert.py create-tfvars --region us-east-1 --stack-name mystack --ssh-key-path /path/to/key.pem > terraform/terraform.tfvars
+
+Note that the ssh key is handled different in the terraform module. With CDK,
+we used an existing keypair, while the terraform module creates a keypair for
+you. The local key does not need to be the one you used with CDK. The new
+autoscaling group nodes and bastion node will be provisioned with the new key.
+
+### Get AWS resources to import
+
+Then, we'll use `convert.py` to generate the Terraform import commands to import our AWS resources:
+
+    ./convert.py get-imports --region us-east-1 --stack-name mystack > terraform/imports.sh
```

Comment:
We need to add `chmod +x terraform/imports.sh` here or prior to executing it below.

Reply:
So the directions here actually do say to just run `bash imports.sh`, so the chmod isn't actually necessary as the instructions are written. I can totally understand reading the instructions and abstracting it into the way you're habituated to operating, though. I could add the chmod in case that happens to anyone else, but on the other hand it adds an extra step. Not sure which is the better option.

Reply:
Valid arguments. I'm okay with moving forward.